backend/test/routes/model/getModel.spec.ts

```diff
@@ -0,0 +1,21 @@
+// Some bad parts
+import '../../../src/utils/test/testUtils.js'
```

Comment:
You shouldn't need this for V2 route unit tests as we have testUtils/routes.ts. To mock the config and user implementation you just need to include the mock in the route test files without any factory and it will use the mocked implementation in the __mocks__ folder.